doc/source/whatsnew/v0.21.0.txt

```diff
@@ -472,6 +472,7 @@ Performance Improvements
 - Improved performance of :meth:`Categorical.set_categories` by not materializing the values (:issue:`17508`)
 - :attr:`Timestamp.microsecond` no longer re-computes on attribute access (:issue:`17331`)
 - Improved performance of the :class:`CategoricalIndex` for data that is already categorical dtype (:issue:`17513`)
+- Improved performance of ``RangeIndex.min`` and ``RangeIndex.max`` by using ``RangeIndex`` properties to perform the computations (:issue:`17607`)
```

Comment:
can u use :meth: here

Reply:
Will using `:meth:` work in this case?  It doesn't look like `RangeIndex` is in `api.rst`, and I can't find any `RangeIndex` related items in the existing HTML API reference.  Not super well versed on how the docs work though, so could be mistaken.

Reply:
so should prob add ``RangeIndex`` to api.rst :>

(and prob any other Indexes that are missing). We don't need to repeat the methods though (they are inherited).